.ci/lumen_cli/cli/lib/core/vllm/vllm_build.py

```diff
@@ -153,18 +159,50 @@ def run(self):
         """
         inputs = VllmBuildParameters()
         logger.info("Running vllm build with inputs: %s", inputs)
-        clone_vllm()
+        vllm_commit = clone_vllm()
 
         self.cp_dockerfile_if_exist(inputs)
-
         # cp torch wheels from root direct to vllm workspace if exist
         self.cp_torch_whls_if_exist(inputs)
 
-        ensure_dir_exists(inputs.output_dir)
+        # make sure the output dir to store the build artifacts exist
+        ensure_dir_exists(Path(inputs.output_dir))
 
         cmd = self._generate_docker_build_cmd(inputs)
         logger.info("Running docker build: \n %s", cmd)
-        run_command(cmd, cwd="vllm", env=os.environ.copy())
+
+        try:
+            run_command(cmd, cwd="vllm", env=os.environ.copy())
+        finally:
+            self.genearte_vllm_build_summary(vllm_commit, inputs)
+
+    def genearte_vllm_build_summary(
+        self, vllm_commit: str, inputs: VllmBuildParameters
+    ):
+        if not gh_summary_path():
+            return logger.info("Skipping, not detect GH Summary env var....")
+        logger.info("Generate GH Summary ...")
+
+        write_gh_step_summary("## Build vllm against Pytorch CI")
+        write_gh_step_summary(
+            f"**Vllm Commit**: [{vllm_commit}](https://github.com/vllm-project/vllm/commit/{vllm_commit})"
+        )
+        torch_sha = os.getenv("GITHUB_SHA")
+
+        if torch_sha:  # only can grab this in github action
+            write_gh_step_summary(
+                f"**Pytorch Commit**: [{torch_sha}](https://github.com/pytorch/pytorch/commit/{torch_sha})]"
+            )
+        vllm_artifact_dir = inputs.output_dir / "wheels"
```

Comment:
I think we could see if the output from https://github.com/vllm-project/vllm/blob/main/vllm/collect_env.py could be used here.  People are familiar with its output